setup.cfg

```
@@ -19,12 +19,15 @@ install_requires =
     celery >= 5.0
     click
     datalad >= 0.18.0
+    datalad-metalad >= 0.4
     Flask
     Flask-SQLAlchemy
     importlib-metadata; python_version < "3.8"
     SQLAlchemy >= 1.4, < 2.0
     psycopg2 >= 2.9, < 3.0
     pydantic >= 1.10
+    datalad-catalog >= 0.2.1, < 0.3
```

I believe it is identical to 
```suggestion
    datalad-catalog ~= 0.2.1
```
right @jwodder ?

---

Yes.